modules/highgui/src/window_gtk.cpp

```diff
@@ -1973,7 +1982,25 @@ static gboolean icvOnMouse( GtkWidget *widget, GdkEvent *event, gpointer user_da
 //        if((unsigned)pt.x < (unsigned)(image_widget->original_image->width) &&
 //           (unsigned)pt.y < (unsigned)(image_widget->original_image->height) )
         {
-            int flags = (state & GDK_SHIFT_MASK ? CV_EVENT_FLAG_SHIFTKEY : 0) |
+            int flags = 0;
+            if (state & GDK_SCROLL_MASK) {
+                cv_event = CV_EVENT_MOUSEMOVE;
+                int orient = CV_EVENT_MOUSEWHEEL;
+                switch(event->scroll.direction) {
+                case GDK_SCROLL_RIGHT: orient = CV_EVENT_MOUSEHWHEEL;
+                case GDK_SCROLL_UP:    flags |= ((1 << 16) | orient);
+                    break;
+                case GDK_SCROLL_LEFT:  orient = CV_EVENT_MOUSEHWHEEL;
+                case GDK_SCROLL_DOWN:  flags |= ((-1 << 16) | orient);
```

Comment:
Looks like there is different units for CV_GET_WHEEL_DELTA macro in Win32 and GTK backends.
Win32 backend uses GET_WHEEL_DELTA_WPARAM which is "expressed in multiples or divisions of WHEEL_DELTA, which is 120".

Do you know what units is used in [gdk-event-get-scroll-deltas](https://developer.gnome.org/gdk3/stable/gdk3-Events.html#gdk-event-get-scroll-deltas) call?